content/en/docs/hertz/tutorials/framework-exten/log.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Logger Extension"
+linkTitle: "Logger Extension"
+date: 2022-06-22
+weight: 2
+description: >
+
+---
+
+Hertz provides logger extension, and the interface is defined in `pkg/common/hlog`. 
+
+## Interface Definition
+
+In Hertz, the interfaces `Logger`, `CtxLogger`, `FormatLogger` are defined in `pkg/common/hlog`, and these interfaces are used to output logs in different ways, and a Control interface is defined to control the logger.
+If you'd like to inject your own logger implementation, you must implement all the above interfaces (i.e. FullLogger). Hertz already provides a default implementation of `FullLogger`.
+
+```go
+// FullLogger is the combination of Logger, FormatLogger, CtxLogger and Control.
+type FullLogger interface {
+   Logger
+   FormatLogger
+   CtxLogger
+   Control
+}
+```
+
+Note that the default logger makes use of the standard library `log.Logger` as its underlying output. So the filenames and line numbers shown in the log messages depend on the settings of call depth. Thus wrapping the implementation of hlog may causes inaccuracies for these two value.
```

Comment:
causes -> cause
two value -> two values

Reply:
Resolved, and updated the same places in Kitext doc